### ISSUE TITLE
Quickstart Failure - Install

### ISSUE THREAD

**Issue by dk4tz** [youtype/mypy_boto3_builder#272]
**Describe the bug**
After installing VSCode Extension and enabling Pylance + type checking mode in settings.json, we run into an error after hitting th install button. **.../bin/python -m pip install -U "boto3-stubs[dynamodb,lambda]==1.34.109"**

running command manually in terminal produces this error:
ERROR: Could not find a version that satisfies the requirement boto3-stubs==1.34.109 (from versions: 0.1.8, 0.1.10, ...

**To Reproduce**
Install VSCode, enable Pylance + type checking and click "Install"

**Additional context**
MacOS 14.5, VSCode 1.89.1, Python 3.10.13

<img width="237" alt="image" src="https://github.com/youtype/mypy_boto3_builder/assets/82852452/c67d09c4-e087-4908-ae1c-d79c50c364d2">

**Comment by vemel** [comment]
Hello! Thank you for the report. I will check and let you know.

**Comment by vemel** [comment]
No, `botocore-stubs` version is not frozen, so any version can be installed.
